docs/layerzero-oft-api-reference.md

```
@@ -0,0 +1,259 @@
+}
+```
+
+---
+
+## extraOptions Encoding (TYPE_3)
+
+Used in `SendParam.extraOptions` to specify gas for `lzReceive` on the destination chain.
+
+### Basic lzReceive option (most common)
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)